app/views/admin/users/show.html.erb

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
If your admin user show page only shows the user's username, you should probably just scrap the show page entirely and just have a list of users with a delete button next to them on the user index page. Either that or flesh out this page to be a little cooler :)